webapp/src/Service/ExternalContestSourceService.php

```diff
@@ -1435,7 +1439,7 @@ protected function importSubmission(Event $event, EventData $data): void
                     'message' => 'No source files in event',
                 ]);
                 $submissionDownloadSucceeded = false;
-            } elseif (($data->files[0]->mime ?? null) !== 'application/zip') {
+            } elseif ($data->files[0]->mime !== null && $data->files[0]->mime !== 'application/zip') {
```

Comment:
FYI: I thought this was also invalid per spec, but indeed in this case the `mime` attribute may be omitted as it is assumed to be `application/zip`, see https://ccs-specs.icpc.io/2023-06/contest_api#submissions